data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
+- What city and country will you reside in during the summer?
+- Time Zone:
+- GSoC blog RSS feed URL: (You will have to write a blog post every
+                          week during GSoC.)
+
+
+Code Sample
+-----------
+
+Link all your contributions to coala and any other open source project here.
+For example, to list your contributions to coala you can mention:
+
+- https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
+for coala repo
+- https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
+for coala-bears repo
+
+You may also mention specific issues/commits using the commit id if you want
+us to take a look at that.
+
+You will have to finish the coala newcomer guide to be accepted:
+http://coala.io/newcomer
+
+
+Project Info
+------------
+
+- Which project from http://projects.coala.io are you applying for?
+If you have your own idea, please add it to our projects list:
+https://github.com/coala/projects/blob/master/data/projects.js
+
+- How is your project helping coala and its community?
+
+- What is the final goal for this project? What would make it a total
+and perfect success?
+
+- Are you already engaged with the project's possible mentors and do
+you have any preference for a particular mentor?
```

Comment:
I'd add something like "We will try to get your choice of a primary mentor. Usually the first listed mentor on a project is intended as the primary one. You will always have at least one primary and one secondary mentor for any project."